lib/util.js

```diff
@@ -0,0 +1,25 @@
+var url = require("url");
+var qs = require("qs");
+
+/**
+ * getNextLink
+ * @function
+ * @param {Object} response A response object returned from calling 'client.makeRequest'
+ * @returns A parsed version of the link to the subsequent page, or null if no such page exists.
+ */
+var getNextLink = function (response) {
```

Comment:
Just curious why we wrote this ourselves instead of using a library? https://github.com/thlorenz/parse-link-header . It could possibly be because we don't want our sdk to have a lot of dependencies which is fair, but then we probably shouldn't use qs either and just stick to core node libraries.